resources/views/peptideSummary.html

```diff
@@ -141,11 +143,17 @@
                 }
             },
             failure: function(response) {
-                Ext4.get(plotPanelId).update("<span class='labkey-error'>Error: " + LABKEY.Utils.encodeHtml(response.exception) + "</span>");
+                showErrorMessage("Error: " + response.exception);
             }
         });
     }
 
+    function showErrorMessage(message) {
+        $('#fom-loading').hide();
+        document.getElementById('fom-error').innerText = message;
```

Comment:
```suggestion
        document.getElementById('fom-error').innerText = LABKEY.Utils.encodeHtml(message);
```

Reply:
Setting `innerText` automatically encodes the text. If this used `innerHTML` then it would need to be encoded.